vuepress/docs/next/tutorials/consume/high-avail-tutorial.md

```diff
@@ -113,73 +109,36 @@ redis-cli -h 10.43.99.198 -p 6379 incr mycounter
-    <artifactId>entando-plugin-jpredis</artifactId>
-    <type>war</type>
-</dependency>
-```
 
-5. Save the pom.
-6. Build and push a custom image for the `entando-de-app` with [Building a Docker Image Tutorial](../devops/build-core-image.md).
-7. Download the operator configuration deployment file:
+## Configure the Implementation
 
-<EntandoCode>curl -L -C - -O https://raw.githubusercontent.com/entando/entando-releases/{{$site.themeConfig.entando.fixpack.v71}}/dist/ge-1-1-6/namespace-scoped-deployment/orig/namespace-resources.yaml > namespace-resources.yaml</EntandoCode>
-
-8. Update the image for `entando-de-app-wildfly` in the `namespace-resources.yaml` file to point to your custom `entando-de-app` image with Redis. The line to change is in the ConfigMap named `entando-docker-image-info`.
-
-9. Deploy your edited file with `kubectl`.
-
-```sh
-kubectl apply -f namespace-resources.yaml
-```
-
-10. Download the `entando-app.yaml` template
+1. Download the `entando-app.yaml` template
 
 <EntandoCode>curl -sLO "https://raw.githubusercontent.com/entando/entando-releases/{{$site.themeConfig.entando.fixpack.v71}}/dist/ge-1-1-6/samples/entando-app.yaml"</EntandoCode>
 
-11. Add environment variables to the `EntandoApp` definition to match your Redis instance. The variables to create are REDIS_ACTIVE, REDIS_ADDRESS (e.g. _redis://localhost:6379_), and REDIS_PASSWORD.
-
->NOTE: This example uses a Secret for the `REDIS_PASSWORD`, which is recommended. You can also hardcode the password in the YAML for testing purposes, but the use of clear text passwords in deployment files is not recommended. **Create and use a Secret for the password as a best practice.**
-
+2. Add these environment variables to the `EntandoApp` YAML to match your Redis instance. The variables to create are `REDIS_ACTIVE`, `REDIS_SESSION_ACTIVE`, `REDIS_ADDRESS` (e.g. _redis://localhost:6379_), and `REDIS_PASSWORD`.
 ```yaml
 data:
   environmentVariables:
     - name: REDIS_ACTIVE
       value: "true"
     - name: REDIS_ADDRESS
-      value: YOUR-REDIS-URL
+      value: YOUR-REDIS-URL 
+    - name: REDIS_SESSION_ACTIVE 
+      value: "true"
     - name: REDIS_PASSWORD
       valueFrom:
         secretKeyRef:
           key: password
           name: YOUR-REDIS-SECRET-NAME
           optional: false 
 ```
+Both REDIS_ACTIVE and REDIS_SESSION_ACTIVE need to be set to "true" to enable the storage of HTTP sessions. If only REDIS_ACTIVE is set to "true", Redis is used just for the cache.
+
+>NOTE: This example uses a Secret for the `REDIS_PASSWORD`, which is recommended. You can also hardcode the password in the YAML for testing purposes, but the use of clear text passwords in deployment files is not recommended. **Create and use a Secret for the password as a best practice.**
 
-12. Deploy your file
+3. Deploy your file
 ```sh
 kubectl apply -f entando-app.yaml
 ```
+You now have an activated Redis implementation for a highly available cluster of Entando. 
```

Comment:
odd wording... "You now have an active Redis implementation for an Entando cluster with high availability" - "highly available" does not appear to be used in lieu of "high availability" (which is commonly/consistently used).

Redis implementation **for**... confirming that this is an implementation **of** Redis **for** the cluster ??